webpack/ForemanInventoryUpload/Components/InventoryFilter/InventoryFilterReducer.js

```diff
@@ -37,7 +37,7 @@ export default (
     }
     case LAYOUT_INITIALIZE: {
       // Layout action changed in Jul 20 2020 - https://github.com/theforeman/foreman/commit/e4c39a7d8f8b50ba45ef63e46f6f6914b69f247a
-      const { title } = organization; // org was renamed
+      const { title } = organization || {}; // org was renamed
```

Comment:
The plugin pages didn't load without adding this as I was getting `organization=undefined`